integration/spark/build.gradle

```diff
@@ -116,10 +135,16 @@ task javadocJar(type: Jar, dependsOn: javadoc) {
     from javadoc.destinationDir
 }
 
-test {
-    useJUnitPlatform {
-        excludeTags 'integration-test'
+task bumpVersionTestFiles() {
+    doLast {
+        def dir = new File(project.projectDir.path + '/integrations')
+        dir.eachFileRecurse (FileType.FILES) { file ->
+            ant.replace(file: file, token: "\$VERSION", value: project.version.toString())
+        }
     }
+}
```

Comment:
Please let's not do this. I removed this functionality before because it clutters the git workspace- every time I have to remember to restore these files after running the build. Let's just do the find/replace in the tests when we read these files.